pkg/crc/network/utils.go

```diff
@@ -121,7 +123,7 @@ func CheckCRCLocalDNSReachableFromHost(clusterName, domainName, appsDomainName,
 		 * in this case, /etc/resolver/ will not be used, so we won't
 		 * have wildcard DNS for our domains
 		 */
-		appsHostname := fmt.Sprintf("foo.%s", appsDomainName)
+		appsHostname := bundle.GetAppHostname("foo")
```

Comment:
:+1:

Reply:
If you think it's better this way, I'll squash the `fixup` commit

Reply:
@cfergeau yes I think it much better, I will have to test this tomorrow on mac.